### DISCUSSION TITLE
Got the repo compiling!

### DISCUSSION THREAD

**Discussion by 0xLPircy** [firefly/.github#1] [General]
i got the pixie-firmware repo compiled (with too many hours of debugging), used component-scene and component-display for firefly-scene and firefly-display.
But ngl too scared to flash it into my device... i like how it is working, i dont know if the one ive done will work. Can anyone help?

i added back the function setNodePosition in firefly-scene directory
rest changes are in pixie-firmware fork i made.
https://github.com/0xLPircy/pixie-firmware

**Comment by ricmoo** [comment]
Heya!

Sorry for the commits to component-scene that weren't published yet. They have now been pushed live.

You can use [this tool](https://firefly.box/demo-provision/) to back up the various parts of your device; the most important thing to not lose is the attest partition, as this contains your encrypted attestation key. Currently only Chrome is supported. Let me know if you have any issues using it. A better tool is coming. :)

**Reply by 0xLPircy** [reply]
hey the link youve attached seems to be privateHey the link youve attached seems to be private

**Reply by ricmoo** [reply]
Sorry. Seems to have been a copy-paste error. Fixed now. :)

**Reply by zmrp** [reply]
can i work on this site and flash part, pretty excited with this mini device ?
i just flashed my device for the first time, would love to explore more on what can be built.

**Reply by ricmoo** [reply]
Can you send a screen shot? Someone else send a screenshot and it looks like it wasn't loading parts of the CSS. Is the text white (which it is supposed to be) or black (which is basically illegible).

I also need to add the latest firmware to the dropdown.

The site is very preliminary, but I'll add the site shortly (it will be in the `pixie-repl` repo) so you can try messing around with it. :)

**Reply by ricmoo** [reply]
The site source code is now available [here](https://github.com/firefly/pixie-repl/tree/wip-v0.1/web). :)

**Reply by ankitzm** [reply]
the text was black but after refreshing it changed to white/gray
thanks, will take up thing from my personal account @ankitzm :)
